c/termination-restricted-15-todo/MinusMin_true-termination.c

```diff
@@ -11,13 +11,14 @@ int main() {
     y = __VERIFIER_nondet_int();
     res = 0;
     
-    if (y < x) { min = y; }
-    else { min = x; }
+    if (x < y) { min = x; }
+    else { min = y; }
     
     while (min == y) {
         y = y+1;
-        if (y < x) { min = y; }
-        else { min = x; }
+        res = res+1;
```

Comment:
When I noticed that there may be an overflow either way, I reversed my changes and and moved this example into todo. So this is the original version.